cypress/integration/Stepper._spec.js

```diff
@@ -0,0 +1,163 @@
+describe("Stepper.vue", () => {
+    it("renders correctly", () => {
+        cy.visit("http://localhost:8081/ ");
```

Comment:
@obulat  The problem that appears here is that in my local machine the port redirects itself to  8081

Reply:
Can you check if something else is running on 8080?

Reply:
Did you read the link I sent you? 

I would like the baseUrl to be set in the json cypress settings file, so that then we don't have to repeat the address in each test, which is prone to errors.  After these changes, we can just write `cy.visit('/')` instead of writing the URL in each test.